### PR TITLE
added beta program page

### PR DESCRIPTION
**Pull request by orianapagerduty**
Hackday by Amanda, Oriana and Ryan.
Updated dev. site with beta portal page.

### CODE REVIEW

**Comment by divtxt** [comment]
:+1: - screenshot?

File: source/documentation/beta/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        
+      </th>
+    </tr>
+</table>
+<br />
+<br />  
+<br />  
+
+<p>
+  Feel free to email us if you have any questions at beta@pagerudy.com.
```
Comment (AmyC) [inline source/documentation/beta/index.html]: Typo in email address. Also, you might want to make it a `mailto:` link.

**Comment by orianapagerduty** [comment]
Thank you @divtxt and @AmyC I made the correction and added the suggestions.

**Comment by orianapagerduty** [comment]
@divtxt and @AmyC - I updated it please review.

**Comment by orianapagerduty** [comment]
screenshot 2015-04-17 16 40 16
screenshot 2015-04-17 16 08 52

**Comment by divtxt** [comment]
LGTM

**Comment by AmyC** [comment]
:100:

**Pull request merged**